Infisical/infisical
# feat: secret sharing

# Description 📣
Adds the ability to share secrets in a time-sensitive manner from the Infisical web dashboard with anybody (non-Infisical users as well) securely using a signing algorithm.

## Loom Walkthrough
https://www.loom.com/share/cb54a37ae5724b5f85026425fe366e5a?sid=15e15f84-d447-4cf1-b5ef-90ac109c8621

## Type ✨


# Tests 🛠️
- Head over to any organisation's and project that you have access to
- Visit the *Secret Sharing* tab from the left menu
- On the Top Right, click on *Add Secret* & fill in the details
- Now visit the URL you get after submitting the details in an incognito tab to access the secret
- See that the secret gets expired after the time allowed by the creator both in the dashboard as well as the viewer

---
